controllers/common/util.go

```diff
@@ -308,3 +309,54 @@ func findNamespace(nsList []string, nsName string) (exist bool) {
 	}
 	return
 }
+
+// CheckSaas checks whether it is a SaaS deployment for Common Services
+func CheckSaas(r client.Reader) (enable bool, err error) {
+	cmName := constant.SaasConfigMap
+	cmNs := "kube-public"
+	saasConfigmap := &corev1.ConfigMap{}
+	err = r.Get(context.TODO(), types.NamespacedName{Name: cmName, Namespace: cmNs}, saasConfigmap)
+	if errors.IsNotFound(err) {
+		return false, nil
+	} else if err != nil {
+		return false, err
+	}
+	v, ok := saasConfigmap.Data["ibm_cloud_saas"]
+	if !ok {
+		return false, nil
+	}
+	if v != "true" {
+		return false, nil
+	}
+	return true, nil
+}
+
+// GetControlNs gets control namespace of deploying cluster scope services
+func GetControlNs(r client.Reader) (controlNs string) {
+	// default control namespace
+	controlNs = constant.ControlNamespace
```

Comment:
I set the default value for `controlNs` to `ibm-cs-control` temporarily.
We could have a discussion about choosing a proper name on the sync up meeting.

Also I firstly set `controlNs` to `MasterNamespce` here https://github.com/IBM/ibm-common-service-operator/blob/36a20289b90b930091ac98b30a719be61a41b112/controllers/bootstrap/init.go#L115
then if it is not a SaaS deployment, all the services will be deployed into `MasterNamespace`.